guides/common/modules/con_hammer-cli-overview.adoc

```diff
@@ -0,0 +1,26 @@
+[id="Hammer-CLI-Overview_{context}"]
+= Hammer CLI Overview
+
+Hammer is a command-line tool to configure and manage a {ProjectServer} through CLI commands or automation in shell scripts.
+Hammer also provides an interactive shell. 
+
+Benefits of using Hammer:
+
+* Shell scripts created for Hammer allow for basic task automation.
+* Output from Hammer commands can be redirected to other tools.
+* The `--debug` option can be used to test responses to API calls before applying them in a script, for example, `hammer --debug organization list`.
+* Changes in the API are automatically reflected in Hammer, while scripts using the API directly have to be updated manually.
+
+To issue Hammer commands, a user must have access to the system on which {ProjectServer} is running.
+
+NOTE: The {ProjectWebUI} has development priority.
+Some features that are available through {ProjectWebUI} might not be available for Hammer.
```

Comment:
This reads strangely for me. I would try to find some other wording, with hopefully a more positive statement.

Reply:
Addressed in https://github.com/theforeman/foreman-documentation/pull/2607/commits/b640ede2acfc629e9807f0f716af97783ab1270c

Reply:
Much better!